src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.Android.cs

```diff
@@ -40,26 +40,9 @@ private static void SetSoftwareRendering(View nativeView, bool enabled)
 		/// <inheritdoc />
 		private protected override bool IsSourceReady => _buffer != null;
 
-		[StructLayout(LayoutKind.Explicit)]
-		private struct ByteArrayToIntArrayBridge
-		{
-			[FieldOffset(0)]
-			public byte[] ByteArray;
-
-			[FieldOffset(0)]
-			public int[] IntArray;
-
-#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
-			public ByteArrayToIntArrayBridge(byte[] bytes)
-#pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
-			{
-				ByteArray = bytes;
-			}
-		}
-
 		private static ImageData Open(byte[] buffer, int bufferLength, int width, int height)
 		{
-			var bitmap = Bitmap.CreateBitmap(new ByteArrayToIntArrayBridge(buffer).IntArray, width, height, Bitmap.Config.Argb8888!);
+			var bitmap = Bitmap.CreateBitmap(MemoryMarshal.Cast<byte, int>(buffer).ToArray(), width, height, Bitmap.Config.Argb8888!);
```

Comment:
Why is the struct problematic? The motivation here was to avoid allocating a new array for performance reasons.

Reply:
Sharing pointers is not the intent of the memory aliasing, it's generally done to alias values from the struct. It is likely that in this context, the pointers were incorrectly moved/deallocated while calling native code. It may be because the memory was not pinned, but for now it needs to not crash first.